adapters/impactify/impactify.go

```diff
@@ -0,0 +1,198 @@
+package impactify
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+	"strings"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+type ImpactifyExtBidder struct {
+	Impactify openrtb_ext.ExtImpImpactify `json:"impactify"`
+}
+
+type DefaultExtBidder struct {
+	Bidder openrtb_ext.ExtImpImpactify `json:"bidder"`
+}
+
+func (a *adapter) MakeRequests(bidRequest *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errs []error
```

Comment:
The `errs` and `validImps` variables are unused since you return immediately from any error encountered while processing an impression. Did you intend for invalid impressions to be filtered out instead?

Reply:
No we don't intend to filter impressions, i remove the two useless variables.